README.md

```diff
@@ -36,6 +36,12 @@ Repo: [github.com/NebraLtd/hm-gatewayrs](https://github.com/NebraLtd/hm-gatewayr
 
 This container is the actual Helium Gateway-rs software (from upstream), with the required configuration files added.
 
+Gatewayrs container requires two IP ports exposed as default.
```

Comment:
we will have to see [this](https://github.com/helium/gwmp-mux) repo's impact on ports etc. No actions required. I have created a [new ticket](https://github.com/NebraLtd/helium-miner-software/issues/422) just to investigate later

Reply:
:+1: